flake.nix

```
@@ -92,7 +95,7 @@
     juno-src.url = github:CosmosContracts/juno/v13.0.1;
 
     osmosis-src.flake = false;
-    osmosis-src.url = github:osmosis-labs/osmosis/v15.0.0;
+    osmosis-src.url = github:osmosis-labs/osmosis/v15.2.0;
```

```suggestion
    osmosis-src.url = github:osmosis-labs/osmosis/v16.1.0;
```
Would it be possible to update Osmosis to `v16.1.0`?

---

did osmosis upgraded to this on mainnet?

---

if they deployed and we can see it - sure, assuming wasmd deps also ok. 

did they updated wasmd dep in this release? if yes, than cannot change like this, need wasmd fix too.

---

i did minor update which works with all scripts for 15 , major bump can bring breaking changes to local envs of 15. i am not doing it in this PR as long as 15 works well.

---

in place to 16 needs to test module, which i am not up to. while 15 likely works